cli/waiter/subcommands/delete.py

```diff
@@ -27,7 +26,7 @@ def delete_token_on_cluster(cluster, token_name, token_etag):
                     print_error('\nPlease kill these services before deleting the token they rely on.')
                 return False
         else:
-            print_error(response_message(resp.json()))
```

Comment:
Should we log the response?

Reply:
It will get logged in the `http_util.get` call, which will show up if you run with `--verbose`